python/ray/rllib/dqn/dqn.py

```diff
@@ -168,12 +181,22 @@ def step(self, cur_timestep):
             self.episode_lengths.append(0.0)
         return ret
 
+    def collect_samples(self, num_steps, cur_timestep):
```

Comment:
why not keep `do_steps` with a flag to `store` for replay buffer, otherwise no storing
and always returning all states?

Reply:
`def do_steps(self, num_steps, cur_timestep, store=False):`

Reply:
Done